src/pykeen/training/accelerated_training_loop.py

```diff
@@ -0,0 +1,420 @@
+# -*- coding: utf-8 -*-
```

Comment:
@cthoyt is this file now obsolete?

Reply:
yes but I don't want to delete it until @migalkin is all done experimenting